augly/image/intensity.py

```diff
@@ -100,6 +100,14 @@ def crop_intensity(metadata: Dict[str, Any], **kwargs) -> float:
     return resize_intensity_helper(metadata)
 
 
+def distort_barrel_intensity(**kwargs) -> float:
+    return 100.0
```

Comment:
Can you define this in terms of a, b, c, & d params? Does changing them really not change the intensity of the augmentation?

Reply:
Well, different parameter set obviously affects the "intensity" of the augmentation, however, I did not come up with a practical intensity function as the parameters are not bounded from above nor below. Here, I think the intensity is something that is defined heuristically here, so I don't think of an intensity representation of such distortion. If you have any alternative, I can change the function accordingly. 

Having said that, my first thought was to determine an upper (and lower) bound for intensity, and also we know that if those parameters adds up to 1 it means no change, so we maybe we want to implement this as an intensity ?